try :
    num = input("Enter a number: ")
    print(int(num))

except Exception as e:
    print(e)

else:
    print("inside else")

    